.gitignore

```diff
@@ -1,3 +1,4 @@
+rg-client-test
```

Comment:
Why dow e ignore this? It contains sources afaict

Reply:
I don't want to commit a binary

Reply:
changes to the cli/ directory will be shown:
```
% echo foo >> cli/rg-client-test/main.go                  update/k8s
[sszuecs@sandor-lab:~/go/src/github.com/szuecs/routegroup-client]% git st                                                  update/k8s
On branch update/k8s
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   cli/rg-client-test/main.go

```